recipes/conversation_assistant/conversation.js

```diff
@@ -48,6 +54,66 @@ const tjConfig = {
 //     servoPin: 7
 // };
 
+async function converse(message) {
+    let assistantSessionId;
+
+    // set up the session if needed
+    if (!assistantSessionId) {
```

Comment:
... it will always go down the `true` path here, right? is this something we need to do once and then cache somewhere, or something we do on every call to `converse()`? (iirc, I think we just do this once and cache it?)